### ISSUE TITLE
[BUG] Random tags on posts

### ISSUE THREAD

**Issue by KHannibal** [sunt-programator/CodeIT#212]
My website can be found here : https://chl.li/fc

## Describe the bug

I have migrated from LoveIt to CodeIt. Everything is running perfectly but the tags of each post appear to be randomly generated, then one tag is chosen and applied to all the posts. See pictures below.

Categories are not affected. I have not altered any file in the `/themes/CodeIT` folder.

When looking at the `/public/tags` folder, the index.html and index.xml files seem to be ok and have the adequate content.

## Expected behavior

Normal behavior regarding tags. :)

## Screenshots

Here is how tags are displayed on my website.

<img width="406" alt="Screenshot 2021-05-31 à 16 33 09" src="https://user-images.githubusercontent.com/3843004/120208581-eec2e900-c22d-11eb-99cd-da7a8f546d93.png">

Just for the sake of clarity, the .md file of the concerned article contains the following line with the four tags:

<img width="664" alt="Screenshot 2021-05-31 à 16 35 53" src="https://user-images.githubusercontent.com/3843004/120208900-4eb98f80-c22e-11eb-938f-1a81c2f5079f.png">

## Build Environment

- OS: macOS Big Sur
- Theme version : release 0.1.2
- Hugo version : v0.83.1+extended darwin/arm64

## Preview Environment

- OS: idem
- Browser : Brave (same issue with Safari, Firefox, etc.)

## Additional Information

Front matter code below

`title: ""
subtitle: ""
date: 2020-12-30
slug: ""
tags: ["xxx","yyy"]
categories: ["zzz"]
featuredImage: "/images/aaa.jpg"
featuredImagePreview: "/images/aaa.jpg"
images: ["/images/aaa.jpg"]
rssFullText: true
hiddenFromHomePage: false
hiddenFromSearch: false
toc:
  enable: true
  auto: false
share:
  enable: true
comment:
  enable: true
show_comments: true
draft: false
`

**Comment by victor-pogor** [comment]
Hi @KHannibal,

Sorry for this late response, I did not start yet to reproduce this issue, I will look at it ASAP. Thank you for your contributions.

**Comment by KHannibal** [comment]
> Hi @KHannibal,
> 
> Sorry for this late response, I did not start yet to reproduce this issue, I will look at it ASAP. Thank you for your contributions.

Thank you @victor-pogor.

The issue is still there with hugo v0.85.0+extended darwin/arm64. I could not find the source of the issue. Here are some more information : 

- my "tags" folder structure and content
![Screenshot 2021-07-08 à 07 43 55](https://user-images.githubusercontent.com/3843004/124868659-581de080-dfc0-11eb-86f1-f1e91f3a667f.png)

- I have removed `auto: true` (or `auto: false`) from the front matter code.

- I have made no change to the theme files, besides removing a few lines in the `archetypes/default.md` file in the root repository and adapting the french translation in the `i18n` folder.
